docs/src/views/FocusTrapZone.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import { link, code } from '../utils/helpers'
+
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Focus Trap Zone">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/focus-trap-zone#overview')}</li>
+      <li>{link('Usage', '/focus-trap-zone#usage')}</li>
+      <li>
+        {link(
+          'Override FocusTrapZone settings',
+          '/focus-trap-zone#override-focustrapzone-settings',
+        )}
+      </li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      FocusTrapZone grabs the focus and traps it within an HTML element, usually a dialog or popup.
+      Pressing TAB key will circle focus within the inner focusable elements of the FocusTrapZone.
+      The main purpose is to block user interaction outside FocusTrapZone in any way. Therefore,
+      keyboard events are not propagated outside FocusTrapZone.
+    </p>
+    <p>
+      Stardust leverages Focus Trap Zone component which is based on the{' '}
```

Comment:
lets be consistent with the component's name, and replace 'Focus Trap Zone' with `FocusTrapZone`